src/lib/dashboard/TopCards.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+	import { Avatar, Card, CardActions, ProgressLinear } from 'svelte-materialify';
+	import { topCards } from './data';
+</script>
+
+{#each topCards as card}
```

Comment:
Please avoid this loop/each. I would like dedicated component for each card (same as example-material-ui)